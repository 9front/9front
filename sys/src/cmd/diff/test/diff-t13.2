A line of text
Another line of text
..
.
A third line
...
..
We keep counting
.
