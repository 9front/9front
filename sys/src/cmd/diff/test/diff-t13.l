A line of text
.
Another line of text
..
A third line
...
A fourth line
.
We keep counting
.
.
