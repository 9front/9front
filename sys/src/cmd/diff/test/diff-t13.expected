--- diff-t13.l
+++ diff-t13.r
@@ -1,11 +1,9 @@
 A line of text
-.
 Another line of text
 ..
+.
 A third line
 ...
-A fourth line
-.
+..
 We keep counting
-.
 .
